platform/linux-generic/odp_timer.c

```diff
@@ -1861,20 +1875,30 @@ void odp_timer_print(odp_timer_t timer)
 	idx = handle_to_idx(timer, tp);
 	tim = &tp->timers[idx];
 
-	_ODP_PRINT("\nTimer info\n");
-	_ODP_PRINT("----------\n");
-	_ODP_PRINT("  timer pool     %p\n", (void *)tp);
-	_ODP_PRINT("  timer index    %u\n", idx);
-	_ODP_PRINT("  dest queue     0x%" PRIx64 "\n", odp_queue_to_u64(tim->queue));
-	_ODP_PRINT("  user ptr       %p\n", tim->user_ptr);
-	_ODP_PRINT("  periodic ticks %" PRIu64 "\n", tim->periodic_ticks & ~PERIODIC_CANCELLED);
-	_ODP_PRINT("\n");
+	len += _odp_snprint(&str[len], n - len, "Timer info\n");
+	len += _odp_snprint(&str[len], n - len, "----------\n");
+	len += _odp_snprint(&str[len], n - len, "  handle         0x%" PRIx64 "\n",
+			    odp_timer_to_u64(timer));
+	len += _odp_snprint(&str[len], n - len, "  timer pool     %p\n", (void *)tp);
```

Comment:
`odp_timer_pool_print()` does not seem to print this anymore, only the handle. Is it cumbersome to map a timer/timeout to a pool based on the prints now?

Patch 25b32cb commit log could be improved with a few words about why `_ODP_PRINT()` is swapped to `_odp_snprint()`.

Reply:
Both fixed in V2. Added also timer handle print to `odp_timer_print ()`.